ush/HWP_tools.py

```diff
@@ -25,6 +38,22 @@ def check_restart_files(hourly_hwpdir, fcst_dates):
     return(hwp_avail_hours, hwp_non_avail_hours)
 
 def copy_missing_restart(nwges_dir, hwp_non_avail_hours, hourly_hwpdir):
+    """Check the restart files.
+
+    Args:
+         nwges_dir: holds the boundary, INPUT, and RESTART files (e.g,
+         described here:
+         [ufs-community/ufs-srweather-app#654](https://github.com/ufs-community/ufs-srweather-app/issues/654))
+         The use of NWGES is a bit of a legacy variable name in
+         production. It is often used to define a directory holding
+         DA-relevant files (such as first guess and analysis files)
+         outside of the main COM output directory.
+         hwp_non_avail_hours: a list of hours without available RESTART files
+         hourly_hwpdir: ???

```

Comment:
@jordanschnell do you know what parameter hourly_hwpdir is?

Reply:
hourly_hwpdir is the directory where the RESTART data is copied (nwges/HOURLY_HWP)